src/solver/conv_winoRxS.cpp

```diff
@@ -948,19 +950,43 @@ ConvSolution ConvBinWinogradRxSf2x3g1::GetSolution(const ConvolutionContext& ctx
     return tunable.GetSolution(ctx, problem, tunable.GetDefaultPerformanceConfig(ctx, problem));
 }
 
-bool ConvBinWinogradRxSf2x3g1Fused::IsApplicable(const ConvolutionContext&) const
+namespace fusion {
```

Comment:
I would suggest moving fused solver to a separate file.

Reply:
Done